Framework/Parallel/inc/MantidParallel/IO/EventParser.h

```diff
@@ -146,80 +126,94 @@ void EventParser<TimeOffsetType>::setEventDataPartitioner(
   m_partitioner = std::move(partitioner);
 }
 
-namespace detail {
-/** Uses MPI calls to redistribute chunks which must be processed on certain
- * ranks.
- * @param comm MPI communicator.
- * @param result output data which must be processed on current rank. (May
- * be updated by other ranks)
- * @param data Data on this rank which belongs to several other ranks.
- */
+/** Set the unit of the values in `event_time_offset`.
+ *
+ * The unit is used to initialize a scale factor needed for conversion of
+ * time-of-flight to microseconds, the unit used by TofEvent. */
 template <class TimeOffsetType>
-void redistributeDataMPI(
-    Communicator &comm, std::vector<Event<TimeOffsetType>> &result,
-    const std::vector<std::vector<Event<TimeOffsetType>>> &data) {
-  using Event = Event<TimeOffsetType>;
-  if (comm.size() == 1) {
-    result = data.front();
+void EventParser<TimeOffsetType>::setEventTimeOffsetUnit(
+    const std::string &unit) {
+  constexpr char second[] = "second";
+  constexpr char microsecond[] = "microsecond";
+  constexpr char nanosecond[] = "nanosecond";
```

Comment:
I suppose it's reasonable to assume that case will not be an issue we would have to consider?

Reply:
What case?

Reply:
A few lines below, where the comparison is made (sorry should have put my comment there) on the `==` comparisons. i.e. we are being strict about file authors not having "Microseconds" instead of "microseconds"

Reply:
I am not sure how precise the standard is formulated in that respect. Changing this in case it causes issues should be trivial in the future, I think.